### PR TITLE
Input issue fixes

### PR DESCRIPTION
**Pull request by cnolimit**

### CODE REVIEW

File: src/components/InputMobileNumber.js [hunk 1]
```diff
@@ -217,7 +217,7 @@ class InputMobileNumber extends Component {
 
   render() {
     return (
-      <SContainer>
+      <SContainer {...this.props}>
```
Comment (IljaDaderko) [inline src/components/InputMobileNumber.js]: Maybe spread in render to follow similar pattern?

File: src/components/InputPassword.js [hunk 2]
```diff
@@ -91,7 +91,7 @@ class InputPassword extends Component {
   render() {
     const { requirementColour, requirements, ...props } = this.props;
     return (
-      <div>
+      <div {...props}>
```
Comment (IljaDaderko) [inline src/components/InputPassword.js]: div can't have any props

Comment (cnolimit) [inline src/components/InputPassword.js]: By that do you mean a plain `<div />` can't have props but a `<Container />` which is just `styled.div` can?

File: src/components/InputMobileNumber.js (hunk 1 above)
Comment (cnolimit) [inline src/components/InputMobileNumber.js]: not sure what you mean, its using the spread operator. unless you mean destruct it firs `const = { ...props } = this.props` ??

**Pull request merged**